docs/service_table.md

```diff
@@ -10,7 +10,7 @@ View [Service Graph](./v1Graph.md) to visualize request flow.
 | [currencyservice](../src/currencyservice/README.md)             | C++             | Converts one money amount to another currency. Uses real values fetched from European Central Bank. It's the highest QPS service.            |
 | [emailservice](../src/emailservice/README.md)                   | Ruby            | Sends users an order confirmation email (mock).                                                                                              |
 | [featureflagservice](../src/featureflagservice/README.md)       | Erlang/Elixir   | CRUD feature flag service to demonstrate various scenarios like fault injection & how to emit telemetry from a feature flag reliant service. |
-| [frontend](../src/frontend/README.md)                           | Go              | Exposes an HTTP server to serve the website. Does not require signup/login and generates session IDs for all users automatically.            |
+| [frontend](../src/frontend/README.md)                           | JavaScript/TypeScript              | Exposes an HTTP server to serve the website. Does not require signup/login and generates session IDs for all users automatically.            |
```

Comment:
Maybe just JavaScript? (considering the [OpenTelemetry JavaScript](https://github.com/open-telemetry/opentelemetry-js#about-this-project) name), so we don't struggle with JavaScript/TypeScript/ECMAScript/CoffeeScript https://coffeescript.org/.